### DISCUSSION TITLE
feat: `bat` rename

### DISCUSSION THREAD

**Discussion by Maik93** [tshakalekholoane/bat#95] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**

This application overlaps to a different executable: https://github.com/sharkdp/bat an enhanced cat command.

**Describe the solution you'd like**

Is there somehow a way to instruct the builder to generate a different target executable?

**Comment by tshakalekholoane** [comment, marked as answer]
You can modify the following line in the Makefile or just use `mv` to rename it after building it.

https://github.com/tshakalekholoane/bat/blob/bd230b88e19839a31056721e8d9907c74c4e7946/Makefile#L24

```diff
- GOOS=linux GOARCH=amd64 go build -ldflags=$(FLAGS) -o=./bin/bat .
+ GOOS=linux GOARCH=amd64 go build -ldflags=$(FLAGS) -o=./bin/your_preferred_name .
```

Similarly, if you're using `make install`, change any occurrence of `bat` under that Make target to your preferred name.

https://github.com/tshakalekholoane/bat/blob/bd230b88e19839a31056721e8d9907c74c4e7946/Makefile#L26-L32

**Comment by pepa65** [comment]
This `bat` is of very limited application, to users of a particular type of laptop, and in a very different domain than the bat-replacement-of-cat that is found in all kinds of repos. If repo maintainers would choose to include this app, they surely will give their package a different name.
